packages/cli/src/voting/revealVotes.js

```diff
@@ -30,6 +31,8 @@ const revealVotes = async (web3, oracle, designatedVoting) => {
     );
   } else if (filteredRequests.length === 0) {
     console.log("No pending votes to reveal!");
+  } else if (round.snapshotId.toString() === "0") {
+    console.log("Snapshot must be taken before reveal");
```

Comment:
I added this to cli app to warn the user before reveal to do a snapshot, but i did not add the functionality to do the snapshot, or really test this. If more work is needed i suggest a new card specifically for cli app.

Reply:
added an issue https://github.com/UMAprotocol/protocol/issues/1988